src/Sentry/Platforms/Native/buildTransitive/Sentry.Native.targets

```diff
@@ -17,12 +17,12 @@
   <ItemGroup Condition="$(TargetFramework.StartsWith('net8')) and '$(OutputType)' == 'Exe' And '$(RuntimeIdentifier)' == 'linux-x64'">
     <DirectPInvoke Include="sentry-native" />
     <NativeLibrary Include="$(MSBuildThisFileDirectory)..\sentry-native\linux-x64\libsentry-native.a" />
-    <LinkerArg Include="-lcurl"/>
+    <NativeSystemLibrary Include="curl" />
```

Comment:
Ha. That's the gotcha?

Reply:
see https://github.com/dotnet/runtime/issues/97414 for details, just added it to the description too